tests_require_fixes/utils/resolver_runner.py

```
@@ -0,0 +1,46 @@
+
+from collections.abc import Callable, Mapping
+
+import pytest
+
+
+def make_resolver_test(
+    *,
+    resolver: Callable[[str], str],
+    data: Mapping[str, str],
+    test_name: str = "test_resolver_data",
+    marks: list[pytest.MarkDecorator] = [],
```

Ruff (0.14.14)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)